test/apps/test_fixtures.clj

```diff
@@ -31,33 +30,6 @@
   (or (io/resource resource-name)
       (throw+ {:message (str "resource " resource-name " not found")})))
 
-(defn- load-ontology [username]
-  (let [ontology-file     (getenv "APPS_ONTOLOGY_FILE" default-ontology-file)
-        ontology-location (find-resource ontology-file)]
-    (->> (http/post (str (curl/url (config/metadata-base) "admin" "ontologies"))
-                    {:query-params     {:user username}
-                     :multipart        [{:part-name "ontology-xml"
-                                         :name      ontology-file
-                                         :mime-type "application/xml"
-                                         :content   (io/file ontology-location)}]
-                     :follow-redirects false})
-         :body
-         service/parse-json
-         :version)))
-
-(defn- delete-ontology [username ontology-version]
-  (http/delete (str (curl/url (config/metadata-base) "admin" "ontologies" (curl/url-encode ontology-version)))
-               {:query-params     {:user username}
-                :follow-redirects false}))
-
-(defn with-ontology [f]
-  (let [username         "ipctest"
-        ontology-version (load-ontology username)]
-    (cp/add-hierarchy-version username ontology-version)
-    (f)
-    (delete :app_hierarchy_version (where {:version ontology-version}))
-    (delete-ontology username ontology-version)))
-
```

Comment:
Loading the ontology in a test fixture turned out to be extremely brittle, so I removed the need for it.